src/components/_common/contact-option/opening-info/helpers/iconUtils.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import style from '../../ContactOption.module.scss';
+
+const appOrigin = process.env.APP_ORIGIN;
+
+type UseHoverAndFocus = [
+    boolean,
+    {
+        onMouseEnter: () => void;
+        onMouseLeave: () => void;
+        onFocus: () => void;
+        onBlur: () => void;
+    },
+];
+
+export function useHoverAndFocus(): UseHoverAndFocus {
+    const [isHovered, setIsHovered] = useState(false);
+    const [isFocused, setIsFocused] = useState(false);
+
+    const hoverAndFocusHandlers = {
+        onMouseEnter: () => setIsHovered(true),
+        onMouseLeave: () => setIsHovered(false),
+        onFocus: () => setIsFocused(true),
+        onBlur: () => setIsFocused(false),
+    };
+
+    return [isHovered || isFocused, hoverAndFocusHandlers];
```

Comment:
Dette er tror jeg bare er en smakssak, men personlig liker jeg best når hooks returnerer et objekt som jeg kan destructe slik at det ikke er noen fare for at rekkefølgen fucker opp. Jeg vet at useState returnerer en Array, og det er det vel delte meninger om.

Kun et innspill altså, og siden det er to parametre så er det jo ikke superfarlig. Hadde vært viktigere hvis det feks hadde vært 3 eller mer...

Reply:
Er egentlig veldig enig, vet ikke helt hva jeg tenkte 🙈